tests/test-files/good/Elm-0.19/AllSyntax/Expressions.elm

```diff
@@ -273,6 +273,16 @@ lambdaWithMultilinePattern =
 ifStatement =
     let
         a =
+            if True then 1 else 2
```

Comment:
For completeness, I think it would be good to add a single-line else if example, assuming that's allowed:

```
if True then 1 else if True then 2 else 3
```

Reply:
Oh, nevermind, from the implementation code, that's not allowed to be single-line.

Reply:
Yeah, it seems to me that `else if` were not part of the use cases listed in the discussion and lack readability for single-line, I have therefore forbidden them.

That's also why I have added the `tests/test-files/transform/Elm-0.19/IfElseIfAlwaysMultiLine.elm` transform test.